assets/js/steps/utils/banner-adapter.js

```diff
@@ -19,40 +19,30 @@ export function adaptListOfBooksToBanner(listOfBooks, reservedMaterialInstructio
     const items = [];
 
     listOfBooks.forEach((book) => {
-        const displayInfo = { ...book };
-
-        if (book.title) {
-            displayInfo.text = `${book.title}`;
-        } else if (book.author && book.title) {
-            // @TODO: Translatable text.
-            displayInfo.text = `${displayInfo.title} af ${book.author}`;
-        } else {
-            displayInfo.text = book.itemIdentifier;
-        }
-
+        const displayInfo = { ...book, text: book.title && book.author ? BannerTitleAuthor(book.title, book.author) : '' };
         switch (book.status) {
             case BookStatus.ERROR:
                 displayInfo.title = book.message;
                 break;
             case BookStatus.IN_PROGRESS:
                 displayInfo.title = BannerAdapterFetchingInfo;
+                displayInfo.text = book.itemIdentifier;
                 break;
             case BookStatus.CHECKED_IN:
             case BookStatus.CHECKED_OUT:
             case BookStatus.RENEWED:
-                displayInfo.title = book.title;
-
-                if (book.reservedByOtherUser) {
+                if (book.author) {
+                    displayInfo.text = BannerTitleAuthor('', book.author);
+                }
+                if (book.message === 'Reserveret') {
```

Comment:
Why not use book.reservedByOtherUser which is set in the engine?

Reply:
I did not know this! I Will!